tests/smoke.rs

```diff
@@ -10,6 +13,13 @@ use walkdir::WalkDir;
 
 static TESTDATA_DIR: &str = "testdata";
 
+#[cfg(unix)]
+const MAILDIR_NAME: &str = "maildir-unix";
+#[cfg(windows)]
+const MAILDIR_NAME: &str = "maildir-windows";
```

Comment:
Here I just copied the folder and renamed the file with the semicolon `;` separator.

Reply:
I'd prefer to not duplicate test data. Can we instead just rename from ':' to ';' where appropriate when testing on Windows? Already the `with_maildir` function is copying the test data to a tmp dir for running the tests; after the percent-decoding step we can do a replacement of ':' to ';' and then use the modified name when copying to the tmp dir. Let me know if that doesn't make sense.

Reply:
:heavy_check_mark: